strawberryfields/apps/qchem/dynamics.py

```diff
@@ -117,3 +122,92 @@ def op(t, Ul, w, q):
         sf.ops.Interferometer(Ul) | q
 
     return op
+
+
+def sample_fock(
+    input_state: list,
+    t: float,
+    Ul: np.ndarray,
+    w: np.ndarray,
+    n_samples: int = 1,
```

Comment:
The number of samples also should probably not have a default value, but always be specified